### PR TITLE
Load unpacked Phar Lap EXP data into memory

### PR DESCRIPTION
**Pull request by slartibardfast**
adds LoadImage an equivalent to UnpackImage

### CODE REVIEW

File: src/ImageLoaders/MzExe/PharLapExtender.cs [hunk 1]
```diff
@@ -72,7 +72,10 @@ public override Program LoadProgram(Address? addrLoad)
             if ((fileHeader.flags & FlagImagePacked) != 0)
             {
                 UnpackImage(fileHeader, image);
+            } else {
```
Comment (uxmal) [inline src/ImageLoaders/MzExe/PharLapExtender.cs:75]: Consider adhering to the coding format in the code base: `{` opens a new line always.

**Review by uxmal — COMMENTED** [review]
Thanks for the contribution! I've just one tiny nitpick re. formatting. Also, because you're a first time contributor, please add yourself to `AUTHORS` in $(RekoDir).

File: src/ImageLoaders/MzExe/PharLapExtender.cs (hunk 1 above)
Comment (slartibardfast) [reply src/ImageLoaders/MzExe/PharLapExtender.cs:75]: oh cool yes, I'll fix that and the whitespace as soon as I can tonight.

Thanks very much!

Comment (slartibardfast) [reply src/ImageLoaders/MzExe/PharLapExtender.cs]: fixed now! thanks again

-dc

**Comment by uxmal** [comment]
Thanks for the contribution!

**Pull request merged**